ionic2/config.xml

```diff
@@ -1,61 +1,62 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This is a file that was carried over by RC0 I think. I don't think the current version uses this file.